test/typing/pass/creation_ops.py

```diff
@@ -0,0 +1,112 @@
+# flake8: noqa
+import torch
+from torch.testing._internal.common_utils import TEST_NUMPY
+if TEST_NUMPY:
+    import numpy as np
+
+# From the docs, there are quite a few ways to create a tensor:
+# https://pytorch.org/docs/stable/tensors.html
+
+# torch.tensor()
+torch.tensor([[0.1, 1.2], [2.2, 3.1], [4.9, 5.2]])
+torch.tensor([0, 1])
+torch.tensor([[0.11111, 0.222222, 0.3333333]],
+                         dtype=torch.float64,
+                         device=torch.device('cuda:0'))
```

Comment:
minor: funky indentation here. also further down in multiple places. maybe you had an editor setup issue? would be good to clean up